src/main/resources/application.yml

```diff
@@ -37,24 +45,27 @@
 
   services:
     persistence:
+      baseUrl: http://localhost:8282
       #service.url:  ${probate_persistence_service_url}:${probate_persistence_service_port}/formdata  #Till byron fixed side way communcation of app
-      formdata.url:  http://localhost:8282/formdata
+      formdata.url:  ${services.persistence.baseUrl}/formdata
       #service.url:  ${probate_persistence_service_url}:${probate_persistence_service_port}/submissions  #Till byron fixed side way communcation of app
-      submissions.url:  http://localhost:8282/submissions
-      sequenceNumber.url:  http://localhost:8282/sequence-number
+      submissions.url:  ${services.persistence.baseUrl}/submissions
+      sequenceNumber.url:  ${services.persistence.baseUrl}/sequence-number
 
     coreCaseData:
-      url: https://case-data-app.dev.ccd.reform.hmcts.net:4451/citizens/%s/jurisdictions/PROBATE/case-types/GrantOfRepresentation
+      baseUrl: https://case-data-app.dev.ccd.reform.hmcts.net:4451
+      url: ${services.coreCaseData.baseUrl}/citizens/%s/jurisdictions/PROBATE/case-types/GrantOfRepresentation
       enabled: ${USE_CCD:false}
 
   auth.provider.service.client:
-    baseUrl: http://localhost:4502
+    #baseUrl: http://localhost:4502
+    baseUrl: http://betadevaccidams2slb.reform.hmcts.net
```

Comment:
I don't think we should change this. 
It's overridden when it's deployed, and probably used when testing locally with the stub.